components/portfolio/Balances.vue

```diff
@@ -23,17 +23,16 @@
         :balance="balance"
         :total-rewards-per-denom="totalRewardsPerDenom"
         :send="true"
+        @open-send-modal="openSendModal(balance.denom)"
       />
     </TableContainer>
-    <LazySendModal ref="SendModal" :denoms="getAllDenoms" />
+    <SendModal ref="SendModal" :denoms="getAllDenoms" />
```

Comment:
why not lazy`?

Reply:
i thought it was interfering with events - but it wasn't - i will revert